src/script.ts

```diff
@@ -54,72 +33,143 @@ export const runScript = async (
       'k6'
     )
   } else {
-    // only the architecture directory will be in resources on the packaged app
     k6Path = path.join(process.resourcesPath, getArch(), 'k6')
   }
 
   // add .exe on windows
   k6Path += getPlatform() === 'win' ? '.exe' : ''
 
-  const k6Args = [
-    'run',
-    modifiedScriptPath,
-    '--vus=1',
-    '--iterations=1',
-    '--insecure-skip-tls-verify',
-    '--log-format=json',
-    '--quiet',
-    ...(!enableUsageReport ? ['--no-usage-report'] : []),
-  ]
-
-  // Create symlink to data folder so it can be accessed by the script
-  await symlink(DATA_FILES_PATH, DATA_FILES_TEMP_PATH, 'dir')
```

Comment:
According to [this thread](https://github.com/nodejs/node/issues/18518#issuecomment-1509547477), it's not possible to create symlinks with `fs.symlink` unless the user is an admin